internal/events/flagged_driver.go

```diff
@@ -0,0 +1,162 @@
+//
+// Copyright 2023 Stacklok, Inc.
```

Comment:
I hate to be the person who is commenting on copyright headers of all things in a PR, but isn't this going to make our CI choke on the header (and if it's not maybe it should..)

Reply:
We don't have anything in CI to choke on this, and I'm not sure we should, because then it would need to understand or choke on renames.  (But then again, I'm not sure we need the date in the copyright header at all, between the Berne convention and various other sources of that information, like `git`.)